tests/requirements.txt

```diff
@@ -32,3 +32,6 @@ pytest-xdist
 requests
 responses
 ruamel_yaml
+# needed for conda-libmamba-solver downstream tests
+conda-forge::mamba >=0.19.0
+conda-forge::libcurl
```

Comment:
@jaimergp Do we still need this?

Reply:
I guess we can install these at runtime downstream if needed.

Reply:
Yeah, I think that'd be good to separate concerns since we don't need the added complexity in the test dependencies here

Reply:
In that case then go ahead and remove these lines and other references to mamba in the CI (if there are any left). Sorry I don't have laptop access to github today for some reason.

Reply:
```suggestion
```